### ISSUE TITLE
Got Error decoder_feature_nan!!! and decoder_feature_inf with hubconf.py

### ISSUE THREAD

**Issue by MENG-G** [YvanYin/Metric3D#121]
I am using the hubconf.py from Metric 3D for inference and using the demo images from the code repository. However, I found that the inference results are completely incorrect. Additionally, I’m encountering the following error messages:
```
decoder_feature_nan!!!
torch.Size([1, 1024, 44, 76])
decoder_feature_inf!!!
torch.Size([1, 1024, 44, 76])
ref_feat_nan!!!
prob_feat_nan!!!
d_nan!!!
norm_nan!!!
abs_rel_err: nan
```
I have checked the code in mono/model/decode_heads/RAFTDepthNormalDPTDecoder5.py. There are no invalid values in the vit_feature output from the encoder. However, after encoder_features = self.token2feature(vit_features), a lot of invalid values appear. I have tried all the ViT models and encountered this problem with each of them. 

Have you come across this issue before?

**Comment by JUGGHM** [comment]
Have you loaded the pre-trained weights? @ZachL1 Could you please help check the correctness of hubconf models?

**Comment by ZachL1** [comment]
I tested it, and everything seems to be working fine. How did you use the sample code? Can you provide more information?

**Comment by djdhdjj** [comment]
I have encountered the same problem. Have you solved it? @ZachL1

**Comment by djdhdjj** [comment]
I've encountered the same problem. It works fine when I use the giant2 version, but this error occurs when I use the large version. May I ask how to solve it? @ZachL1 @JUGGHM 

![Image](https://github.com/user-attachments/assets/2c0fae3f-fb9c-48b8-98d1-a2646fd64c06)

**Comment by djdhdjj** [comment]
https://github.com/YvanYin/Metric3D/issues/121#issuecomment-2650872699
Sorry, it was my mistake. When importing the model, I wrongly wrote the parameter 'pretrain' of the function as 'pretrained'.

**Comment by hhhddddddd** [comment]
@MENG-G Hello, I encountered the same problem as you. How did you solve it?

**Comment by lee12332** [comment]
> [#121 (comment)](https://github.com/YvanYin/Metric3D/issues/121#issuecomment-2650872699) Sorry, it was my mistake. When importing the model, I wrongly wrote the parameter 'pretrain' of the function as 'pretrained'.

me too,thanks a lot
